<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AverageMesh</class>
 <widget class="qMRMLWidget" name="AverageMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>442</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <widget class="QWidget" name="inputFromSceneTab">
      <attribute name="title">
       <string>Input from Scene</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Excluded</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="qMRMLSubjectHierarchyTreeView" name="excludedTree">
         <property name="contextMenuEnabled">
          <bool>false</bool>
         </property>
         <property name="editMenuActionVisible">
          <bool>false</bool>
         </property>
         <property name="levelFilter">
          <stringlist notr="true"/>
         </property>
         <property name="nodeTypes">
          <stringlist notr="true">
           <string>vtkMRMLModelNode</string>
          </stringlist>
         </property>
         <property name="hideChildNodeTypes">
          <stringlist notr="true"/>
         </property>
         <property name="visibilityColumnVisible">
          <bool>false</bool>
         </property>
         <property name="idColumnVisible">
          <bool>false</bool>
         </property>
         <property name="colorColumnVisible">
          <bool>false</bool>
         </property>
         <property name="transformColumnVisible">
          <bool>false</bool>
         </property>
         <property name="includeItemAttributeNamesFilter">
          <stringlist notr="true"/>
         </property>
         <property name="includeNodeAttributeNamesFilter">
          <stringlist notr="true"/>
         </property>
         <property name="excludeItemAttributeNamesFilter">
          <stringlist notr="true"/>
         </property>
         <property name="excludeNodeAttributeNamesFilter">
          <stringlist notr="true"/>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="excludeButton">
           <property name="text">
            <string>↑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="includeButton">
           <property name="text">
            <string>↓</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Included</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="qMRMLSubjectHierarchyTreeView" name="includedTree">
         <property name="contextMenuEnabled">
          <bool>false</bool>
         </property>
         <property name="editMenuActionVisible">
          <bool>false</bool>
         </property>
         <property name="levelFilter">
          <stringlist notr="true"/>
         </property>
         <property name="nodeTypes">
          <stringlist notr="true">
           <string>vtkMRMLModelNode</string>
          </stringlist>
         </property>
         <property name="hideChildNodeTypes">
          <stringlist notr="true"/>
         </property>
         <property name="visibilityColumnVisible">
          <bool>false</bool>
         </property>
         <property name="idColumnVisible">
          <bool>false</bool>
         </property>
         <property name="colorColumnVisible">
          <bool>false</bool>
         </property>
         <property name="transformColumnVisible">
          <bool>false</bool>
         </property>
         <property name="includeItemAttributeNamesFilter">
          <stringlist notr="true"/>
         </property>
         <property name="includeNodeAttributeNamesFilter">
          <stringlist notr="true"/>
         </property>
         <property name="excludeItemAttributeNamesFilter">
          <stringlist notr="true"/>
         </property>
         <property name="excludeNodeAttributeNamesFilter">
          <stringlist notr="true"/>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="inputFromFilesTab">
      <attribute name="title">
       <string>Input from Files</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="1">
        <widget class="ctkDirectoryButton" name="inputDirectoryButton"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Inputs</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="outputMeshComboBox">
       <property name="nodeTypes">
        <stringlist notr="true">
         <string>vtkMRMLModelNode</string>
        </stringlist>
       </property>
       <property name="hideChildNodeTypes">
        <stringlist notr="true"/>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
       <property name="interactionNodeSingletonTag">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Average Scalars</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="averageScalarsCheckbox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSubjectHierarchyTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLSubjectHierarchyTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>AverageMesh</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputMeshComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AverageMesh</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>includedTree</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AverageMesh</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>excludedTree</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
